packages/interfaces/src/IAppliance.js

```diff
@@ -17,8 +18,11 @@ class IAppliance {
    * @param  {Object} settings        Values to override default appliance settings.
    * @param  {Object} settings.logger The logger to use.
 	 */
-	// eslint-disable-next-line no-unused-vars
 	constructor(settings = {}) {
+		super({
+			...settings,
+			objectMode: true,
```

Comment:
Note that this means an appliance could NOT override objectMode.  I believe that's what we want but wanted to confirm.

Reply:
I think that's right.